cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -998,6 +999,10 @@ pQueryCmd =
         <*> pNetworkId
         <*> pMaybeOutputFile
 
+    pJsonOutput :: Parser JsonOutput
```

Comment:
Can we just default to json when we write to disk? It will reduce the changes we have to make to the code and users generally want JSON output anyways.

Reply:
Done!